limitador/src/storage/redis/scripts.rs

```diff
@@ -38,10 +38,8 @@ pub const BATCH_UPDATE_COUNTERS: &str = "
         if c == tonumber(delta) then
             redis.call('expire', counter_key, ttl)
             redis.call('sadd', limit_key, counter_key)
-            table.insert(res, ttl*1000)
-        else
-            table.insert(res, redis.call('pttl', counter_key))
         end
+        table.insert(res, redis.call('pexpiretime', counter_key))
```

Comment:
This isn't ideal, as `pexpiretime` was introduced in Redis 7... that being said, [elasticache](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/SupportedCommands.html) supports it

Reply:
Not that it matters right now but it's also supported by [memorystore](https://cloud.google.com/memorystore/docs/cluster/supported-commands) and I think Azure cache